src/VirtoCommerce.Platform.Modules/Local/LocalStorageModuleCatalog.cs

```diff
@@ -15,51 +17,52 @@ namespace VirtoCommerce.Platform.Modules
     public class LocalStorageModuleCatalog : ModuleCatalog, ILocalModuleCatalog
     {
         private readonly LocalStorageModuleCatalogOptions _options;
+        private readonly IConnectionMultiplexer _redisConnMultiplexer;
         private readonly ILogger<LocalStorageModuleCatalog> _logger;
+        private readonly string _discoveryPath;
 
-        public LocalStorageModuleCatalog(IOptions<LocalStorageModuleCatalogOptions> options, ILogger<LocalStorageModuleCatalog> logger)
+        public LocalStorageModuleCatalog(IOptions<LocalStorageModuleCatalogOptions> options, IConnectionMultiplexer redisConnMultiplexer, ILogger<LocalStorageModuleCatalog> logger)
```

Comment:
Please remove the tightly coupling from `IConnectionMultiplexer` from LocalStorageModuleCatalog. Please leverage the factory pattern instead, and create proper DistributedLockResource instance from a  factory that is injected in the ctor.